lib/ex_doc/retriever.ex

```diff
@@ -458,6 +464,25 @@ defmodule ExDoc.Retriever do
     {id, id}
   end
 
+  defp module_group(module, group_patterns) do
+    Enum.find_value(group_patterns, fn {group, patterns} ->
+      patterns = List.wrap patterns
+      module_in_patterns(module, patterns) && Atom.to_string(group)
+    end)
+  end
+
+  defp module_in_patterns(module, patterns) do
+    "Elixir." <> module_string = Atom.to_string module
```

Comment:
Do not assume the module name is "Elixir." because of Erlang modules. You should probably use the module id or title.